examples/ci/app/ci/upstash/redis.test.ts

```diff
@@ -80,4 +80,21 @@ describe("redis", () => {
       ).not.toThrow()
     })
   })
+
+  test("should fail if marked as failed", async () => {
+
+    const route = "fail-route"
+    const randomId = `random-id-${nanoid()}`
+    const result = `random-result-${nanoid()}`
+
+    // increment, save and check
+    await redis.increment(route, randomId)
+    await redis.saveResultsWithoutContext(route, randomId, result)
+    await redis.checkRedisForResults(route, randomId, 1, result)
```

Comment:
Why do we check this? We already know it'd work

Reply:
just a checkpoint to make sure that the state is set in redis, and only difference is the `failWithoutContext` method